examples/aws-microk8s/README.md

```diff
@@ -1,3 +1,27 @@
 # AWS MicroK8s Cloud Formation Template
 
 Deploy this via the AWS web console and adjust the created Auto Scale Group.
+
+## Components
+
+### SQS and SNS
+
+The SQS and SNS are used to communicate between the Auto Scale Group and Joule.
+
+### Launch Template
+
+The launch template specifies the type of EC2 instances to be created, as well as their AMI. Note that the AMI may have to be changed across regions.
+
+### Auto Scaling Group
```

Comment:
How would I add an extra command to the snap setup?

Reply:
Yep, good point, I'll add that.